packages/devtools/package.json

```diff
@@ -33,6 +33,10 @@
       "types": "./dist/index.d.ts",
       "import": "./dist/server.js"
     },
+    "workerd": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/server.js"
+    },
```

Comment:
I think the `workerd` condition needs to be first (before `browser`).

Reply:
thanks, you're right 🙂👍